docs/data/advanced-components/overview.md

```diff
@@ -0,0 +1,172 @@
+console warnings.
+You must set the license key before rendering the first component.
+You only need to install the key once in your application.
+
+```jsx
+import { LicenseInfo } from '@mui/x-license-pro';
+
+LicenseInfo.setLicenseKey(
+  'x0jTPl0USVkVZV0SsMjM1kDNyADM5cjM2ETPZJVSQhVRsIDN0YTM6IVREJ1T0b9586ef25c9853decfa7709eee27a1e',
+);
+```
+
+### Security
+
+The check of the license key is done without making any network requests.
+
+The license key is designed to be public, the only thing we ask of licensed users is to not proactively publicize their license key.
+Exposing the license key in a JavaScript bundle is expected.
+
+### Validation errors
+
+If the validation of the license key fails, the component displays a watermark and provides a console warning in both development and production.
+End users can still use the component.
+
+Here are the different possible validation errors:
+
+#### Missing license key
+
+If the license key is missing, the component will look something like this:
+
+<div class="only-light-mode">
+  <img src="/static/x/watermark-light.png" style="width: 653px; margin-bottom: 2rem;" alt="" loading="lazy">
+</div>
+<div class="only-dark-mode">
+  <img src="/static/x/watermark-dark.png" style="width: 645px; margin-bottom: 2rem;" alt="" loading="lazy">
+</div>
+
+> Note that you are still allowed to use the component for [evaluation purposes](#evaluation-trial-licenses) in this case.
+
+#### License key expired
+
+The licenses are perpetual, the license key will work forever with the current version of the software.
+
+However, **access to updates/upgrades** is not perpetual.
+Installing a version of the component released after the license key has expired will trigger a watermark and console message.
+For example, if you purchase a one-year license today, you are not licensed to install a version released two years in the future, but you can optionally update to any version, including major versions, if it's released in the next 12 months.
+
+#### Invalid license key
+
+The license key you have installed is not as issued by MUI.
+
+## Support
+
+### GitHub
+
+We use GitHub issues as a bug and feature request tracker. If you think you have found a bug, or have a new feature idea, please start by making sure it hasn't already been [reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed). You can search through existing issues and pull requests to see if someone has reported one similar to yours.
+
+[Open an issue](https://github.com/mui/mui-x/issues/new/choose)
+
+### StackOverflow
+
+For crowdsourced answers from expert MUI developers in our community.
+StackOverflow is also visited from time to time by the maintainers of MUI.
+
+[Post a question](https://stackoverflow.com/questions/tagged/mui)
+
+### Professional support
+
+When purchasing an MUI X Pro or Premium license you get access to professional support for a limited duration.
```

Comment:
What is the length of the "limited duration"? Is it as long as the license is active?

```suggestion
The MUI X Pro and Premium Plans include access to professional support for as long as the license remains active.
```

Reply:
@oliviertassinari ?